test_quick_sort.py

```diff
@@ -0,0 +1,57 @@
+from quick_sort import quisort
+from random import shuffle
+import pytest
+
+
+def test_rand_quicksort():
+    tlist = range(500)
+    shuffle(tlist)
+    assert tlist != range(500)
+    quisort(tlist)
+    assert tlist == range(500)
+
+
+def test_backward_quicksort():
+    tlist = range(500)
+    tlist.reverse()
+    quisort(tlist)
+    assert tlist == range(500)
+
+
+def test_inorder_quicksort():
+    tlist = range(500)
+    quisort(tlist)
+    assert tlist == range(500)
+
+
+def test_sameval_quicksort():
+    tlist = [3 for _ in range(500)]
+    quisort(tlist)
+    for item in tlist:
+        assert item == 3
+
+
+def test_char_quicksort():
+    initlist = list('abcdefghijklmnopq')
+    tlist = initlist[:]
+    shuffle(tlist)
```

Comment:
This is probably fine, but testing is best when you always are running the same test data every time you run the test. Maybe write a 2nd test that sorts from the same fixed jumbled string every time, or make this one fixed.